cmd/kyma/alpha/create/scaffold/default_cr.go

```diff
@@ -0,0 +1,132 @@
+package scaffold
```

Comment:
Can you move any non cmd files into a internal/scaffold pkg?

Reply:
1. These files (except utils.go) are are extending the command struct. Therefore I think they belong together. There are calls to cmd.CurrentStep as well for logging. 

2. And since both packages would be called the same, referencing would hurt the readability.

3. Also the create module command does it this way.

WDYT? Should I really move these?

Reply:
Yeah we should, but can be done afterwards :) 👍